plugins/health_check/ping.ml

```diff
@@ -0,0 +1,20 @@
+open Lwt.Infix
+
+type t = No_context
+
+let id = "ping"
+
+module Key = Address
+
+module Value = Current.Unit
+
+module Outcome = Current.Unit
+
+let publish No_context job key _ =
+  Current.Job.start job ~level:Current.Level.Dangerous >>= fun () ->
+  Current.Process.exec ~cancellable:true ~job ("", [| "ping"; "-c"; "4"; key.Key.ip|])
```

Comment:
One annoying unportable thing is that this'll only work on `ping` commands that support both IPv4 and IPv6.  On macOS, for example, you have to use `ping6` for IPv6 endpoints.

Reply:
I have updated the code to use `ping6` for IPv6 addresses.  On the Linux machines I checked: Ubuntu, Debian, and Raspbian, this was a symlink to `ping`.